lib/rubocop/cop/layout/empty_lines_around_class_body.rb

```diff
@@ -70,8 +70,11 @@ class EmptyLinesAroundClassBody < Cop
         KIND = 'class'.freeze
 
         def on_class(node)
-          _name, _superclass, body = *node
-          check(node, body)
+          _name, superclass, body = *node
+
+          adjusted_first_line = superclass.first_line if superclass
```

Comment:
Could `superclass.first_line` be moved into the `#check` method? If it won’t cause problems for other users of `EmptyLinesAroundBody`, I actually think it belongs there.

Reply:
Yes. I'm concerned about this.

However, I did not move to `EmptyLinesAroundBody` module because I think that the adjustment on line number indicated by the superclass node is specific to `EmptyLinesAroundClassBody` cop.

When moving it to `EmptyLinesAroundBody` module, I think it is necessary to consider nodes other than the superclass node. So I'd like to consider it when other use cases also require `EmptyLinesAroundBody` module.

Reply:
Ahh, you are right. `EmptyLinesAroundClassBody` is in fact the *only* user of `EmptyLinesAroundBody` that have to be concerned about superclass.